Brokerages/Bitfinex/BitfinexBrokerage.cs

```diff
@@ -35,6 +35,7 @@ namespace QuantConnect.Brokerages.Bitfinex
     /// <summary>
     /// Bitfinex Brokerage implementation
     /// </summary>
+    [BrokerageFactory(typeof(BitfinexBrokerageFactory))]
     public partial class BitfinexBrokerage : BaseWebsocketsBrokerage, IDataQueueHandler
```

Comment:
Nice!
I think `TradierBrokerage` is also missing? Suggest let's review any other missing and fix them together 👍

Reply:
I fixed the `TradierBrokerage` one. Both `PaperBrokerage` and `BacktestingBrokerage` are also lacking a `BrokerageFactory` annotation, but since they don't extend `IDataQueueHandler` I don't think they need one.

Reply:
thanks 👍